dockerfiles/puppeteer-context/mina_daemon_puppeteer.py

```diff
@@ -94,12 +94,13 @@ def start_daemon():
   log("touching /root/daemon-active" )
   Path('daemon-active').touch()
   log("daemon fully started" )
-  # log_event("node_initialized", "node has been initialized")
+  log_event("node_started", "node has been started")
 
 def stop_daemon():
   log("stop_daemon called" )
   global mina_process
   mina_process.send_signal(signal.SIGTERM)
+  log_event("node_stopped", "daemon is being stopped by puppeteer script and is going offline")
```

Comment:
Why are we now receiving this event when the signal is sent? It seems more useful to know when we actually have seen all the processes die; the test executive already knows when the request was sent.

Reply:
You're right this puppeteer event no longer is being used.  I figured I'd leave it for "backwards consistency", if you will.  it doesn't hurt to output the log message, even if it's no longer being used by lucy.

As for knowing when the process dies, that's taken care of by the node_down event which is emitted when the log engine fails to contact the node, so we do have that covered.